spec/services/tag_archiver_spec.rb

```diff
@@ -1,168 +0,0 @@
-require "rails_helper"
-
-RSpec.describe TagArchiver do
```

Comment:
Think we need to keep this spec file but remove the topic bits... some of it is still used ie the mainstream browse stuff